### PR TITLE
Fix update tests by not relying on download.o.o v2

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/38393

Todo:

- [x] Live: https://openqa.opensuse.org/tests/717627#step/consoletest_setup/29 ([verification](http://artemis.suse.de/tests/349))
- [x] Boot to snapshot: https://openqa.opensuse.org/tests/717439 ([verification](http://artemis.suse.de/tests/350))
- [x] `INSTALLONLY`: https://openqa.opensuse.org/tests/717442 ([verification](http://artemis.suse.de/tests/354))
- [x] `zypper_info`: https://openqa.opensuse.org/tests/717677#step/zypper_info/7 ([verification](http://artemis.suse.de/tests/359))
- [x] Do the replacement on image creation: https://openqa.opensuse.org/tests/717472 ([verification](http://artemis.suse.de/tests/355))

### CODE REVIEW

**Comment by asdil12** [comment]
Bad test: https://openqa.opensuse.org/tests/717439

File: tests/console/zypper_info.pm [hunk 1]
```diff
@@ -54,12 +54,11 @@ sub run {
         return;
     }
 
-    prepare_source_repo;
-
     # check for zypper info
     test_package_output;
 
     if (is_sle('>=12-SP2') || is_opensuse) {
+        prepare_source_repo;
```
Comment (rwx788) [inline tests/console/zypper_info.pm]: I wonder if we should remove or at least disable sources repo afterwards...

Comment (asdil12) [reply tests/console/zypper_info.pm]: Pending verification run: http://artemis.suse.de/tests/363

Comment (asdil12) [reply tests/console/zypper_info.pm]: Pending verification run for `zypper_info` in extra_tests: http://artemis.suse.de/tests/378

**Comment by rwx788** [comment]
@asdil12 could you please take a look? Test module was not scheduled there, maybe even never...

**Comment by asdil12** [comment]
--> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5532

For some reason `zypper_info` is not scheduled for me when I clone that job: http://artemis.suse.de/tests/386

**Pull request merged**